posthog/settings/access.py

```diff
@@ -59,6 +59,9 @@
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY: str = os.getenv("SECRET_KEY", DEFAULT_SECRET_KEY)
 
+# SECURITY WARNING: keep the encryption secret key used in production secret!
+ENCRYPTION_SECRET_KEY: str = os.getenv("ENCRYPTION_SECRET_KEY", "sNxC-b8AkSkNqs8nVfUz4_gn89MzuQmtckgmxm81LP8=")
```

Comment:
We _might_ want to add a debug thing here so it only uses a default if in DEBUG otherwise throws?